### DISCUSSION TITLE
hardware keyb duplicate keys question

### DISCUSSION THREAD

**Discussion by molleweide** [kmonad/kmonad#411] [Q&A] [answered]
On my ergodox I have put duplicate `space` keys on both left and right inner thumb so that I can use a layout that mimics
my laptop, ie. big laptop space with two thumbs I mimic this on my ergodox by putting a space on both sides.

1. Can this cause any unwanted side effects when using kmonad? 
2. This means that I only need to use a single space key in my `layout.kbd` right?

**Comment by MuhammedZakir** [comment, marked as answer]
> 1. Can this cause any unwanted side effects when using kmonad?

If you try to remap same buttons (keycode) to different buttons, you may get unexpected results when you press them.

> 2. This means that I only need to use a single space key in my layout.kbd right?

Yes.
